src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -22,28 +21,38 @@ class EnrollmentHistory extends React.Component {
     */
 
     // History explanation box
+    let historyExplanationBox;
     const benefitMaybePending =
-      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement > 0) &&
-      enrollmentData.usedEntitlement === 0;
+      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement.months > 0) &&
+      (enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0);
     const historyMayLookIncorrect = enrollments.length > 0 || benefitMaybePending;
-    const noEnrollmentHistory = enrollments.length === 0 && enrollmentData.usedEntitlement > 0;
-    const historyHeading = noEnrollmentHistory ?
-                           (<h4>You don't have any enrollment history</h4>) :
-                           (<h4>Does something look wrong in your enrollment history?</h4>);
-    const historyExplanation = noEnrollmentHistory ?
-                               (<span>Your enrollment history may not be available if:</span>) :
-                               (<span>Certain enrollments may not be displayed in this history if:</span>);
-    const historyExplanationBox = (historyMayLookIncorrect || noEnrollmentHistory) && (
-      <div className="feature">
-        {historyHeading}
-        {historyExplanation}
-        <ul>
-          <li>Your school made a request to us that's still in process, or</li>
-          <li>You made a request to us that's still in process, or</li>
-          <li>You used or are using your benefit for flight, on-the-job, apprenticeship, or correspondence training</li>
-        </ul>
-      </div>
-    );
+    const noEnrollmentHistory = enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0;
+    const benefitUsedUp = enrollmentData.remainingEntitlement.months === 0 && enrollmentData.remainingEntitlement.days === 0;
+
+    if (noEnrollmentHistory) {
+      historyExplanationBox = (
+        <div className="feature">
+          <h4>You don't have any enrollment history</h4>
+          <span>Your enrollment history may not be available if:</span>
+          <ul>
+            <li>You or your school did not yet make a request to us, <strong>or</strong></li>
+            <li>You or your school made a request that's still in process</li>
+          </ul>
+        </div>
+      );
+    } else if (historyMayLookIncorrect || benefitUsedUp) {
```

Comment:
I asked @rroueche and she said that the only conditions needed to show the blue box for Scenario E are:
veteranIsEligible = 1
activeDuty = 0 OR Delimiting/End Date is not NULL (whichever is easier for you)

Going to update this logic to reflect that.